src/config/active-response.c

```diff
@@ -257,14 +356,25 @@ int ReadActiveResponses(XML_NODE node, void *d1, void *d2)
 
     /* Check if timeout is allowed */
     if (tmp_ar->timeout && !tmp_ar->ar_cmd->timeout_allowed) {
-        mdebug1("Timeout is not allowed");
-        minfo(AR_NO_TIMEOUT, tmp_ar->ar_cmd->name);
+        if (output == NULL) {
+            mdebug1("Timeout is not allowed");
+            minfo(AR_NO_TIMEOUT, tmp_ar->ar_cmd->name);
+        } else {
+            snprintf(message, OS_FLSIZE + 1,
```

Comment:
Calling snprintf with OS_FLSIZE + 1 while passing in a message of OS_FLSIZE length pr longer would cause the function to overwrite the byte after the buffer with '\0'. This is not happening because all of the calls are done with small strings, but I encourage this calls to be changed so that the given length passed into snprintf is OS_FLSIZE, improving code safety for the future.